cfgov/v1/util/ref.py

```diff
@@ -25,7 +25,7 @@
 ]
 
 page_types = [
-    ('activity-log', 'Activity Log'),
+    ('activity-log', 'Recent updates'),
```

Comment:
I'm not entirely clear on what the slug in this tuple corresponds to, but I think it should change to match the new name?

Also, we have some inconsistent capitalization for the full names in this list. Currently everything is title case except for "Administrative adjudication".

Reply:
Any idea where I would test this?

Reply:
Maybe @richaagarwal @chosak knows?

Reply:
This code is unfortunately pretty messy. It relates to page types that can be searched using a Filter Controls block on a filterable page. For example, after this PR, a page can be configured with a Filter Controls module of "Recent updates" page type:

![image](https://user-images.githubusercontent.com/654645/30339745-00b060b4-97be-11e7-9fd0-e908be6f3726.png)

When rendered, this page's filter form then contains the various hardcoded categories associated with that page type (under "Category"):

![image](https://user-images.githubusercontent.com/654645/30339768-1bae9bd8-97be-11e7-9a80-c413793987e1.png)

This means that any existing pages in the database that have a form configured like this already have the hardcoded slug `activity-log`, and so changing the slug here as @Scotchester suggests would break those pages. A simultaneous data migration would be needed to keep those in sync.

Regarding the second item in the tuple: its main function is for display in the Wagtail admin, so correcting capitalization should be okay. There's also some additional functionality (see [here](https://github.com/cfpb/cfgov-refresh/pull/3330/files#diff-6de80da6a880654cf8a657eea06ef20aL169)) that does some manual updating of page type - category mappings, and these need to be in sync. So capitalizing "Administrative adjudication" would need to happen in both places.

@richaagarwal did some nice investigative work in this area and in the past we've discussed converting all of this over to some kind of Django model-backed store, which would make these kind of changes less brittle.

Reply:
Great background @chosak! So if I understand correctly the changes I have made are okay to go in?

Reply:
Yes, the only reason I didn't approve was because of the outstanding TODO of renaming the existing Activity Log page. As long as that doesn't get dropped, then this is 👍 !